dev/attest/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Can we just use a similar strategy to the one we used at the repo root to extend the base config in /dev/configs? Don't want to be copying this stuff around.